directus/Caddyfile

```diff
@@ -0,0 +1,28 @@
+# change to the hostname you want to bind to
+localhost:3000
+
+# change to correct document root
+# in this example config directus is assumed to be root
+# you may need to adjust rewrites according to your own setup
+root /var/www/directus
+
+# Set the path to the php-fpm process.
```

Comment:
Yeah, I've been more critical lately :smile: Sorry.

A suggestion to rephrase to: "Set the base path that accesses Directus on your site and the address to php-fpm" or something.